.github/workflows/ci.yaml

```diff
@@ -123,68 +119,22 @@ jobs:
           ruby: true
           gcp-provider: ${{ secrets.GCP_WORKLOAD_IDENTITY_PROVIDER }}
           gcp-service-account: ${{ secrets.GCP_SERVICE_ACCOUNT }}
```

Comment:
We should probably skip GCP auth in this workflow.

Reply:
Deleting the lines will have the same effect, since the default is an empty string, but if you'd rather explicitly pass `''` that's fine.